README.md

```diff
@@ -1,9 +1,176 @@
-# telestion-extension-template
+# telestion-extension-mavlink
 
-A template for creating new Telestion extensions.
+A Mavlink message parser for the Telestion backend.
 
-This template contains an initialized project structure with configured dependencies and a publishing pipeline.
+## Install
 
-To fetch the dependencies locally you need a Github personal access token with `packages:read` rights.
-Either add your github user name and the token as `GITHUB_ACTOR` and `GITHUB_TOKEN` to your environment variables 
-or fill the `.github.credentials.template`.
+First, add the extension package registry to your gradle repositories:
+
+```groovy
+repositories {
+    maven {
+        name = "GitHubMavlink"
+        url = uri("https://maven.pkg.github.com/wuespace/telestion-extension-mavlink/")
+        credentials {
+            username = System.getenv("GITHUB_ACTOR") == null ? GITHUB_ACTOR : System.getenv("GITHUB_ACTOR")
+            password = System.getenv("GITHUB_TOKEN") == null ? GITHUB_TOKEN : System.getenv("GITHUB_TOKEN")
+        }
+    }
+}
+```
+
+Next, add the extension to the project dependency list:
+
+```groovy
+dependencies {
+    // ...
+    implementation 'de.wuespace.telestion.extension:mavlink:0.4.0'
+    // ...
+}
+```
+
+And synchronize gradle.
+
+Finished!
+
+## Usage
+
+The extension provides different verticles, that you can use and set up in your project `config.json`.
+You will need:
+
+- [Mavlink v2 Validator](https://wuespace.github.io/telestion-extension-mavlink/de/wuespace/telestion/extension/mavlink/ValidatorMavlink2.html)
+- [Mavlink Message Registrar](https://wuespace.github.io/telestion-extension-mavlink/de/wuespace/telestion/extension/mavlink/MavlinkRegistrar.html)
```

Comment:
Does not exist in this repository, yet. This was added in the Daedalus-2 package as some kind of placeholder, until I find the time to properly build this and include a verticle in this repository.
If you want to link it here, we should probably commit it though and make changes later.

Reply:
Hmm, ok.
I think, we need to commit it though, because it's way easier to use a verticle in a configuration to handle registered mavlink message types than do it programmatically. (meaning using the `MessageIndex` in another verticle)
But that's my opinion. What do you think?

Reply:
Yes I think that is indeed a good idea.

Reply:
Done in [`a65d67c` (#37)](https://github.com/wuespace/telestion-extension-mavlink/pull/37/commits/a65d67c22f110ff2e2146cc1a87423f83c8d3a98)